bpf/maps_definition.h

```diff
@@ -30,12 +30,10 @@ struct {
     __uint(pinning, LIBBPF_PIN_BY_NAME);
 } additional_flow_metrics SEC(".maps");
 
-//PerfEvent Array for Packet Payloads
+//Ringbuf for Packet Payloads
 struct {
-    __uint(type, BPF_MAP_TYPE_PERF_EVENT_ARRAY);
-    __type(key, u32);
-    __type(value, u32);
-    __uint(max_entries, 256);
+    __uint(type, BPF_MAP_TYPE_RINGBUF);
+    __uint(max_entries, 1 << 16);
```

Comment:
that's much bigger than the perfevent array, is there a specific reason for that?
Also: ringbuf doesn't ask for a type?

Reply:
on 2dn thought 4k is why too much I dropped that to 256 which should be enough for basic hdrs including v6